src/Microsoft.Extensions.DependencyModel/DependencyContextCsvReader.cs

```diff
@@ -46,7 +50,12 @@ public DependencyContext Read(Stream stream)
                     name: identity.Item2,
                     version: identity.Item3,
                     hash: identity.Item4,
-                    assemblies: packageGroup.Select(l => RuntimeAssembly.Create(l.AssetPath)),
+                    assemblies: packageGroup
+                        .Where(l => l.AssetType == "runtime")
+                        .Select(l => RuntimeAssembly.Create(l.AssetPath)),
+                    nativeAssets: packageGroup
```

Comment:
Can you point me to the right place for this test?

Here? https://github.com/dotnet/cli/blob/rel/1.0.0/test/Microsoft.Extensions.DependencyModel.Tests/DependencyContextBuilderTests.cs